packages/socks-proxy-agent/src/index.ts

```diff
@@ -1,5 +1,5 @@
 import { SocksClient, SocksProxy, SocksClientOptions } from 'socks';
```

Comment:
Looks like we can remove the `tls` import below to make lint happy.

Reply:
Done